modules/compose.mjs

```diff
@@ -16,10 +16,11 @@ import * as rwhAccounts from './accounts.mjs';
 import * as rwhUtils from './utils.mjs';
 
 const positionBeforeBegin = 'beforebegin';
+const positionBeforeEnd = 'beforeend';
 const positionAfterBegin = 'afterbegin';
 const fwdHdrLookupString = '-------- ';
 const plainTextFirstChars = '> ';
-const cleanBlockQuoteStyle = 'border:none !important; padding-left:0px !important; margin-left:0px !important;';
+const cleanBlockQuoteStyle = 'border:none !important; padding-left:0 !important; margin-left:0 !important;';
```

Comment:
@ygoe Initially, it was unitless. The user reported that it did not work with other email clients (Apple Mail, Outlook), so I had to add it.

Reply:
Okay, I'm adding the units back. I didn't have that info. Maybe a comment in the code could make it clear to other readers as well so this information doesn't get lost.

Reply:
Thanks @ygoe